src/service/http/fetcher.ts

```diff
@@ -431,3 +432,32 @@ export const fetchMaintenanceInfo = () =>
   axiosIns(`/statistics/maintenance_info`).then((res: AxiosResponse) =>
     toCamelcase<Response.Wrapper<State.MaintenanceInfo>>(res.data.data),
   )
+
+export const exportTransactions = ({
+  type,
+  id,
+  startDate,
+  endDate,
+  fromHeight,
+  toHeight,
+  tab,
+}: {
+  type: State.TransactionCsvExportType
+  id?: string
+  startDate?: Dayjs
+  endDate?: Dayjs
+  fromHeight?: number
+  toHeight?: number
+  tab: string
+}) =>
```

Comment:
A small detail to note, the `tab` parameter is unnecessary/unclear to an API call because `tab` is about UI, not the HTTP request under the hood.

It could be
```typescript
{
  query: {
    type: 'date' | 'block'
    from: string
    to: string
  }
}
```
or 
```typescript
{
  block?: Record<'from' | 'to', string>,
  date?: Record<'from' | 'to', string>
}
```